src/state/opamAction.ml

```diff
@@ -512,9 +512,19 @@ let remove_package t ?keep_build ?silent nv =
 (* Compiles a package.
    Assumes the package has already been downloaded to [source].
 *)
```

Comment:
I would have written one core function and two exported functions for the two modes, rather than a mode argument; but it's just a a matter of taste, this is fine.